dataverse-persistence/src/main/resources/Bundle_pl.properties

```diff
@@ -2041,10 +2041,20 @@ file.versionDifferences.fileUnrestricted=Niezastrze\u017Cony
 file.versionDifferences.fileGroupTitle=Plik
 
 # File Ingest
-ingest.csv.invalidHeader=Niepoprawny wiersz nag\u0142\u00F3wka. Jedna z kom\u00F3rek jest pusta.
-ingest.csv.lineMismatch=Niezgodno\u015B\u0107 liczby linii odnalezionych w pierwszym i ostatnim przebiegu. W pierwszym przebiegu odnaleziono {0} linii, a w ostatnim {1}.
-ingest.csv.recordMismatch=B\u0142\u0105d odczytu linii {0} w pliku danych: oczekiwano {1} oddzielonych warto\u015Bci, odnaleziono {2}.
+ingest.error.INVALID_HEADER=Niepoprawny wiersz nag\u0142\u00F3wka. Jedna z kom\u00F3rek jest pusta.
+ingest.error.LINE_MISMATCH=Niezgodno\u015B\u0107 liczby linii odnalezionych w pierwszym i ostatnim przebiegu. W pierwszym przebiegu odnaleziono {0} linii, a w ostatnim {1}.
+ingest.error.RECORD_MISMATCH=B\u0142\u0105d odczytu linii {0} w pliku danych: oczekiwano {1} oddzielonych warto\u015Bci, odnaleziono {2}.
```

Comment:
Why not keep the CSV prefix for those few? It seems to have been kept for `EXCEL` & `RTAB` above.